.github/workflows/build_and_upload_artifact.yml

```diff
@@ -0,0 +1,35 @@
+name: Build Bahmni Core
+
+on:
+  push:
+    branches: [ master ]
+
+  pull_request:
```

Comment:
I feel that we don't need to do a build and deploy for all pull requests. Could we only run tests for PRs?

Reply:
Created a separate workflow for PRs